libs/blocks/chart/chart.js

```diff
@@ -85,27 +86,42 @@ const barTooltipFormatter = ({
   `${seriesName}<br />${marker} ${value[x[0]]}${unit} ${name}<i class="tooltip-icon"></i>`
 );
 
-const barSeriesOptions = (seriesData, colors, size, unit) => {
+const tooltipFormatter = (params, unit) => {
+  let tooltip = params[0].name;
+  params.forEach(({
```

Comment:
Good candidate for reduce here, doesn't make much difference, but a good candidate none the less. 
```
let tooltip = patams.reduce((ttip, {...} ) => {
    ttip += `<br />${marker} ${value[y[0]]}${unit} ${seriesName}`;
    return ttip;
}, params[0]?.name);
```
You could even add a bit of logic to tell if the last item has been set, and then append add `tooltip += '<i class="tooltip-icon"></i>'`

No need to change, just food for thought.

Reply:
That is a good thought, but since they're so similar, I won't update